src/iris/ui/static/js/iris.js

```diff
@@ -408,9 +456,23 @@ iris = {
     },
     addNotification: function(event){
       var $step = $(event.target).parents('.plan-step'),
+          notifications = $('.plan-notification');
```

Comment:
the end of this line should be a comma not semicolon. this is causing 'template' to become a global variable.